source/elements/oneVPL/include/onevpl/mfxdefs.h

```diff
@@ -187,7 +187,7 @@ MFX_PACK_BEGIN_USUAL_STRUCT()
 typedef struct {
     mfxU32 Min;  /*!< Minimal value of the range. */
     mfxU32 Max;  /*!< Maximal value of the range. */
-    mfxU32 Step; /*!< Value incrementation step. */
+    mfxU32 Step; /*!< Value incrementing step. */
```

Comment:
I don't think incrementation is a word. Or if it is, it's not what you mean. incrementing probably isn't right either. How about

increment value

Reply:
Agree. That's better.